pkg/proxy/node.go

```diff
@@ -109,3 +111,55 @@ func (n *NodeEligibleHandler) OnNodeDelete(node *v1.Node) { n.HealthServer.SyncN
 
 // OnNodeSynced is a handler for Node syncs.
 func (n *NodeEligibleHandler) OnNodeSynced() {}
+
+// NodeManager handles the life cycle of kube-proxy based on the NodeIPs.
+// Implements the config.NodeHandler interface
+// (ref: https://issues.k8s.io/111321)
```

Comment:
Don't link to an issue (that probably talks about lots of other things as well), just explain what it's doing and why.